<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPlayerDialog</class>
 <widget class="QDialog" name="AddPlayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="5">
      <widget class="QLineEdit" name="rtgFideLineEdit">
       <property name="inputMethodHints">
        <set>Qt::InputMethodHint::ImhDigitsOnly</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="natIdLabel">
       <property name="text">
        <string>National ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QComboBox" name="sectionComboBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="genderLabel">
       <property name="text">
        <string>Gender</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="genderEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="natAutofillButton">
       <property name="text">
        <string>Autofill</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="rtgLineEdit">
       <property name="inputMethodHints">
        <set>Qt::InputMethodHint::ImhDigitsOnly</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="fideAutofillButton">
       <property name="text">
        <string>Autofill</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="birthDateEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="fideIDEdit"/>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="sectionLabel">
       <property name="text">
        <string>Section</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="natIdEdit"/>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="teamsLabel">
       <property name="text">
        <string>Teams</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="6">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="rtgLabel">
       <property name="text">
        <string>National Rating</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="birthdateLabel">
       <property name="text">
        <string>Birthdate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="rtgFideLabel">
       <property name="text">
        <string>FIDE Rating</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="fideIdLabel">
       <property name="text">
        <string>FIDE ID</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="teamsEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="teamsSearchButton">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddPlayerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddPlayerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
